### PR TITLE
[DOC-2242] Queue Intelligence

### PR DESCRIPTION
**Pull request by aimurphy**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [X] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* DOC-2242

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [X] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by aman-harness — APPROVED** [review]

File: docs/continuous-integration/troubleshoot/queue-intelligence.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The Queue Intelligence feature introduces a `queued` state for individual builds. Builds progress through the following states:
+
+* `pending`: Build request created and waiting for a delegate. The maximum timeout for this state is 12 hours.
+* `queued`: Build request queued by a delegate. The maximum timeout for this state is 12 hours. When viewing the build in the UI, this state is indicated by a **Queued license limit reached** message.
+* `running`: The delegate runs a build for each build stage in the pipeline. The maximum timeout for this state is one hour.
+
+## Concurrency and resource limits
+
+You CI license determines the maximum number of concurrent builds you can run. Each account has a specified maximum that applies to all builds on all pipelines in the account. Upon hitting the concurrency limit, builds show the **Queued license limit reached** state in the UI.
```
Comment (rajatharanganath) [inline docs/continuous-integration/troubleshoot/queue-intelligence.md:23]: Typo here. You -> Your

Comment (rajatharanganath) [reply docs/continuous-integration/troubleshoot/queue-intelligence.md:23]: @pranav-joshi-harness - Should we mention the default limit's configured by license type?

File: docs/continuous-integration/troubleshoot/queue-intelligence.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: Queue Intelligence
```
Comment (rajatharanganath) [inline docs/continuous-integration/troubleshoot/queue-intelligence.md:2]: hey @pranav-joshi-harness - Please review the title here. This is for the Pipeline Queuing functionality.

Comment (aimurphy) [reply docs/continuous-integration/troubleshoot/queue-intelligence.md:2]: Discussed name w/ @rustd and went with Queue Intelligence.

File: docs/continuous-integration/troubleshoot/queue-intelligence.md (hunk 1 above)
Comment (aimurphy) [reply docs/continuous-integration/troubleshoot/queue-intelligence.md:23]: I will add a link to the pricing page

**Review by douglas-j-bothwell — APPROVED** [review]

**Pull request merged**